src/Core/Components/Drag/FluentDropZone.razor.css

```diff
@@ -0,0 +1,9 @@
+﻿div[draggable='true'] {
+    cursor: move;
+}
+
+div[dragged-over] {
+    background-color: lightgray;
```

Comment:
Do we need to use a FluentUI CSS variable (and not always `lightgray`) ?

Reply:
Good point. Which one would you recommend?